feature-group-definitions/background-gradients.yml

```diff
@@ -0,0 +1,46 @@
+spec: https://drafts.csswg.org/css-images-4/#gradients
+caniuse:
+  - css-gradients
+  - css-repeating-gradients
+  - css-conic-gradients
```

Comment:
Including conic gradients makes the whole feature newer, but it's all still pre-2020 AFAICT. So let's try this.

Reply:
So this ends up being interoperable on 2020-11-17. Long enough for the wider-support status, though not quite pre-2020.

Reply:
Agreed, let's try it like this. I think this should be ready to merge.